graphql-schemas/schema.yml

```diff
@@ -793,7 +793,7 @@ confs:
   - { name: labels, type: json }
   - { name: name, type: string, isUnique: true, isSearchable: true, isRequired: true }
   - { name: description, type: string }
-  - { name: auth, type: ClusterAuth_v1, isInterface: true }
+  - { name: auth, type: ClusterAuth_v1, isInterface: true, isList: true, isRequired: true }
```

Comment:
I think we should not have `isRequired: true`. Some clusters do not define this, when we import them in app-interface and we do not manage them ourselves. For example `c-rh-c-eph`

Reply:
I would like to have it as required because the qontract-reconcile code is much easier to read. No `auth is not None` or `for a in auth or []` everywhere. I'll add `auth: []` to those clusters without an auth ([see this MR](https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/51932))

Reply:
hmm, I see. Fair enough